### DISCUSSION TITLE
請問更新新版中文化時，需要先還原嗎？

### DISCUSSION THREAD

**Discussion by alvinyeung** [GpointChen/FFXIVChnTextPatch-GP#108] [Q&A] [answered]
請問當每次更新新版本的中文化時，需要先還原再重新安裝中文化嗎？
還是不用還原，直接覆蓋就可以了？

**Comment by GpointChen** [comment, marked as answer]
需要還原喔！

直接更新會讓程式備份到漢化過的檔案，所以除非你有自己額外備份原本的遊戲檔案（000000.win32.* 和 0a0000.win32.*），不然不建議直接更新。
